core-bundle/tests/Fixtures/Functional/Routing/news.yml

```diff
@@ -79,4 +79,14 @@ tl_news_archive:
     - id: 1
       tstamp: 1539616536
       title: Test
-      jumpTo: 3
+      jumpTo: 333
+
+tl_page:
+    - id: 333
+      pid: 1
+      tstamp: 1636915068
+      sorting: 512
+      title: News reader
+      alias: news-reader
+      type: regular
+      published: '1'
```

Comment:
@aschempp I am not sure why the functional tests did not fail before here. The news archive of the functional tests reference a page ID of 3 for the news reader page. However, no such page will actually be created during the functional tests.

_Note:_ I used a completely new page ID, because there would be duplicate primary key errors during the tests if I used `3`.